### ISSUE TITLE
[TextStyle] GoogleFonts usage false negative

### ISSUE THREAD

**Issue by pattobrien** [pattobrien/design_system_lints#5]
Instance creation of a TextStyle from GoogleFonts is not reporting a lint:

```dart
Text(
  'My Plants',
  style: GoogleFonts.notoSans(
    color: Colors.black87,
    fontSize: 20.sp,
  ),
),
```

**Comment by pattobrien** [comment]
Fixed with design_system_lints v0.1.2.
